pkg/controller/factory/base_controller.go

```diff
@@ -74,6 +103,10 @@ func waitForNamedCacheSync(controllerName string, stopCh <-chan struct{}, cacheS
 	return nil
 }
 
+func exitOnError(err error) {
```

Comment:
what's the sense of this func? https://wiki.haskell.org/Eta_conversion

Reply:
this is for unit testing so I can replace this with something that check this func was called :-)

Reply:
got rid of eta :-)